### PR TITLE
fix(all workshops): removed region from console links

### LINKED ISSUE #253: region `us-west-2` used in some links
### What is the problem?

The region `us-west-2` is specified in a few workshop sections. This may cause issues for users that choose to implement the workshops in another region.

<kbd>

region-locked-console-hyperlink

</kbd>

### Environment

  - **CDK CLI Version:** 
  - **Section:** 
  - **Browser:** all
  - **Language:** all

### PR DESCRIPTION
**Pull request by ryparker**
## Summary

This PR removes `us-west-2.` from all links since the region is not specified in the workshop and may cause issues for users that choose to implement the workshops in another region.

<kbd>

region-locked-console-hyperlink

</kbd>

Fixes #253

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Comment by ryparker** [comment]
Need to merge https://github.com/aws-samples/aws-cdk-intro-workshop/pull/256 to fix Semantic Rules Check from blocking the merge button.

**Review by peterwoodworth — DISMISSED** [review]

**Comment by ryparker** [comment]
@peterwoodworth Mind giving this one another 👍🏻 ? I had to rebase off master.

**Review by peterwoodworth — APPROVED** [review]

**Pull request merged**
